web/modules/custom/joinup_community_content/joinup_community_content.module

```diff
@@ -133,6 +133,14 @@ function joinup_community_content_workflow_selector(EntityInterface $entity): ?s
     return NULL;
   }
 
+  $user_id = \Drupal::currentUser()->id();
+  /** @var \Drupal\og\OgMembershipInterface $membership */
+  if ($membership = \Drupal::getContainer()->get('og.membership_manager')->getMembership($parent_entity, $user_id)) {
+    if ($membership->hasRole($parent_entity->getEntityTypeId() . '-' . $parent_entity->bundle() . '-author')) {
+      return 'node:post_moderated';
+    }
+  }
```

Comment:
This is hacky so it is a good idea to explain why it is necessary that we mess with the post_moderated state depending on the membership.

Reply:
I have added a commend though this is not really hacky. This is the whole meaning of the ticket. Having a role that behaves as a post moderated role in any group.